spec/storySpec.js

```diff
@@ -1,19 +1,14 @@
 "use strict";
 var assert = new Assert();
+var story = new Story({"id":"sport/live/2017/jun/22/royal-ascot-2017-buildup-to-the-gold-cup-live","type":"liveblog","sectionId":"sport","sectionName":"Sport","webPublicationDate":"2017-06-22T14:28:41Z","webTitle":"Royal Ascot 2017","webUrl":"https://www.theguardian.com/sport/live/2017/jun/22/royal-ascot-2017-buildup-to-the-gold-cup-live","apiUrl":"https://content.guardianapis.com/sport/live/2017/jun/22/royal-ascot-2017-buildup-to-the-gold-cup-live","isHosted":false})
 
 function hasHeadline() {
-	var headline = "This is a headline!";
+	var headline = "Royal Ascot 2017";
   assert.isEqual(story.headline, headline);
 }
 
-function hasText() {
```

Comment:
Why remove this test? We are still going to need this once we have the summary component?